internal/feature_internal.go

```diff
@@ -98,7 +98,13 @@ func (f *ScriptBasedSoftwareUpdatable) load() {
 }
 
 func (f *ScriptBasedSoftwareUpdatable) process() {
+	f.lock.Lock()
+	if !featureAvailable {
+		f.lock.Unlock()
+		return
```

Comment:
If edge client is not connected, the processing will not start. The problematic scenario:
- run software-update and connect to the Mosquitto broker
- after that run suite-connect

Reply:
Checked, method is only called after suite-connect

Reply:
There is execution flow: paho for thing info -> ditto client to process. No issue at all.